src/TuDa.CIMS.Shared/Entities/Consumables.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace TuDa.CIMS.Shared.Entities;
+
+/// <summary>
+/// An entity representing a consumable item in the chemical inventory.
+/// </summary>
+public record Consumables()
```

Comment:
🔧 Class names should be singular
```suggestion
public record Consumable
```